.readthedocs.yml

```diff
@@ -7,3 +7,6 @@ build:
 
 conda:
   environment: doc/rtd_environment.yml
+
+sphinx:
+  configuration: doc/source/conf.py
```

Comment:
You can ignore this in review, it's covered by https://github.com/metoppv/improver/pull/2093
Included here only so I could build the docs on the web for testing and include a link for the reviewer.